repair/row_level.cc

```diff
@@ -3032,6 +3032,14 @@ class row_level_repair {
                     });
                 }).get();
 
+                // Use the average number of partitions, instead of the sum of
+                // the partitions, as the estimated_partitions. The bigger the
+                // estimated_partitions, the more memory bloom filter for the
+                // sstable would consume.
+                if (!master.all_nodes().empty()) {
+                    _estimated_partitions /= master.all_nodes().size();
```

Comment:
This is still over estimation, no? This assumes repair will bring over all data to each node.